extensions/notebook/package.json

```diff
@@ -166,6 +166,11 @@
         "title": "%title.SQL19PreviewBook%",
         "category": "%books-preview-category%"
       },
+      {
+        "command": "books.hybridjupyterbook",
```

Comment:
I don't think this command is necessary - you should be able to just contribute your command directly to the provider books view from your own extension.

Reply:
This command is redundant, but the Provided Books View won't add any books to its view unless its named explicitly as "books.<bookfoldername>" So I have decided to remove the first command in my extension.